frontend/src/components/Add.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useContext } from "react";
+
+import { Button, Form } from "react-bootstrap";
+import Courses from "./Courses";
+import { Link, useNavigate } from "react-router-dom";
+import { AuthContext } from "./AuthContext";
+
+function Add() {
```

Comment:
As a non-admin user, I wasn't able to access any of the components that provide CRUD access to the `Courses` model.